lms/templates/instructor/instructor_dashboard_2/certificate-bulk-white-list.underscore

```diff
@@ -5,10 +5,11 @@
     </p>
     <form id="bulk-white-list-exception-form" enctype="multipart/form-data">
       <div class="customBrowseBtn">
-        <input disabled="disabled" class="browse-file" placeholder="<%- gettext("Choose File") %>" />
+        <label for="bulk-exception-upload" class="sr"><%- gettext('Choose CSV file') %></label>
```

Comment:
We are hiding the "uglier" one in lieu of a "prettier" one. I guess the original author doesn't like the way default file upload input type="file" fields look.

Do you know if the instructor dashboard going to be overhauled prior to 2 October?